### PR TITLE
updated sentence

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: cookbook/configuration/override_dir_structure.rst [hunk 1]
```diff
@@ -176,8 +176,7 @@ The change in the ``composer.json`` will look like this:
         ...
     }
 
-In ``app/autoload.php``, you need to modify the path leading to the
-``vendor/autoload.php`` file::
+In ``app/autoload.php``, you need to modify the path to the ``autoload.php`` file::
```
Comment (javiereguiluz) [inline cookbook/configuration/override_dir_structure.rst]: I like your proposal ... but in the future I'd like to review all the cookbook articles to be more concise and straight-to-the-point. For example, this phrase could be reworded as:

```
Then, update the path to the ``autoload.php`` file in ``app/autoload.php``::
```

**Comment by WouterJ** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks @OskarStark :)

**Comment by OskarStark** [comment]
:+1: you're welcome!

**Pull request merged**